src/iuse.cpp

```diff
@@ -2010,75 +2010,83 @@ int iuse::sew_advanced(player *p, item *it, bool, const tripoint& )
         return mcopy;
     };
 
-    // TODO: Wrap all the mods into structs, maybe even json-able
+    // @todo: Wrap all the mods into structs, maybe even json-able
     // All possible mods here
-    std::array<std::string, 4> clothing_mods{
+    const std::array<const std::string, 4> clothing_mods{
         { "wooled", "furred", "leather_padded", "kevlar_padded" }
     };
 
     // Materials those mods use
-    std::array<std::string, 4> mod_materials{
+    const std::array<const std::string, 4> mod_materials{
```

Comment:
If the array itself is `const`, the type in it doesn't need to be, because constness of array will be passed down to its types when its used (unless you have pointers in the array, but you don't have them here). You can also make the arrays `static`.
```
static const std::array<std::string 4> array_name_here {
```